static/app/views/performance/transactionSummary/transactionReplays/index.tsx

```diff
@@ -0,0 +1,138 @@
+import {Location} from 'history';
+
+import * as Layout from 'sentry/components/layouts/thirds';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t} from 'sentry/locale';
+import {Organization, Project} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {isAggregateField} from 'sentry/utils/discover/fields';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import withOrganization from 'sentry/utils/withOrganization';
+import withProjects from 'sentry/utils/withProjects';
+
+import PageLayout, {ChildProps} from '../pageLayout';
+import Tab from '../tabs';
+
+import ReplaysContent from './content';
+
+type Props = {
+  location: Location;
+  organization: Organization;
+  projects: Project[];
+};
+
+function TransactionVitals(props: Props) {
+  const {location, organization, projects} = props;
+
+  return (
+    <PageLayout
+      location={location}
+      organization={organization}
+      projects={projects}
+      tab={Tab.Replays}
+      getDocumentTitle={getDocumentTitle}
+      generateEventView={generateEventView}
+      childComponent={ReplaysContentWrapper}
+    />
+  );
+}
+
+function ReplaysContentWrapper(props: ChildProps) {
+  const {location, organization, eventView, transactionName, setError} = props;
+
+  return (
+    <DiscoverQuery
+      eventView={eventView}
+      orgSlug={organization.slug}
+      location={location}
+      setError={error => setError(error?.message)}
+      referrer="api.performance.transaction-summary"
```

Comment:
`"api.performance.transaction-summary"` is that something to be updated?

Reply:
@ryan953 I'm not sure honestly. It seems like it's correct since it's 'referrer' and transaction-summary is the parent view but I could be wrong.

Reply:
To me `referrer` means the page that this discovery query is on, which is `/organizations/${orgSlug}/performance/summary/replays/` a new place. Then the backend can tell where each query came from.

Looks like the api has an allow-list of what it'll keep, and stomps on other values: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_events.py#L140-L144
We could need to update that list, looks like `api.performance.transaction-spans` is a similar valid value.

Reply:
Should we just nix that prop for now then? Only reason it's there really is because I was copying stuff out of the similar transaction views

Reply:
naw, it's ok. I think i figured out the scope of it, and we'll be back here again soon for the new api calls.